systemtest/src/test/java/io/strimzi/systemtest/UserST.java

```diff
@@ -80,6 +80,8 @@ void testUserWithNameMoreThan64Chars() {
                 "InvalidResourceException",
                 "True",
                 "NotReady");
+
+        KafkaUserResource.deleteKafkaUserWithoutWait(kafkaUser);
```

Comment:
These methods should not be here for my POV same applied with the topics etc.

Reply:
yes, I'm gonna delete it